src/platform/update/version.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_VERSION_COMPARITOR_H
+#define MULTIPASS_VERSION_COMPARITOR_H
```

Comment:
Bad def.

Reply:
I don’t know whether it’s copying a name/convention from elsewhere, but the usual spelling is compar**a**tor. [[Oxford](https://en.oxforddictionaries.com/definition/comparator), [Merriam-Webster](https://www.merriam-webster.com/dictionary/comparator), [Wikipedia](https://en.wikipedia.org/wiki/Comparator)]

Reply:
@matthewpaulthomas yep you're right

Reply:
file gone